packages/@webex/widget-cc-user-state/src/user-state/user-state.presentational.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import {IUserState} from './use-state.types';
+
+const UserStatePresentational: React.FunctionComponent<IUserState> = (props) => {
+  return (
+    <>
+      <h1 data-testid="user-state-heading">{props.name}</h1>
```

Comment:
Why do we have this `data-testid` everywhere? 

Are we also setup for the UI testing?

Reply:
This I've added for simple unit testing for UI. This is used in the test that checks if the correct prop is rendered. I think that much testing for a basic UI is okay? What do you think?